src/test/server/store-test.js

```diff
@@ -1,9 +1,203 @@
-const UserStore = require('../../../_compiled_/stores/user-store').default;
+const assert = require('assert');
+
+const usersData  = require('../../test/data/users.json');
+const UserAction = require('../../../_compiled_/actions/user-action').default;
+const UserInfo   = require('../../../_compiled_/stores/user-info').default;
+const UserStore  = require('../../../_compiled_/stores/user-store').default;
+const EVENT_TYPE = require('../../../_compiled_/stores/event-type').default;
+
+const TEST_USER = new UserInfo(
+    {
+        name:    'Ervin New',
+        phone:   '89500418181',
+        email:   'vika.chernookaya@mail.ru',
+        website: 'github.com'
+    }
+);
+
+const TEST_USER_EXISTING = new UserInfo(
+    {
+        name:    'Ervin Howell',
+        phone:   '0418181',
+        email:   'ervin@mail.ru',
+        website: 'github.com'
+    }
+);
+
+const SEARCH_USER_INFO = new UserInfo({ name: 'Ervin' });
+
+const SEARCH_NON_EXIST_USER_INFO = new UserInfo({ name: 'Ivan' });
+
+const UPDATE_USER = new UserInfo(
+    {
+        id:      2,
+        name:    'Vika',
+        phone:   '555',
+        email:   'ervin@mail.ru',
+        website: 'github.com'
+    }
+);
+
+const SEARCH_FIELDS = {
+    name:    true,
+    phone:   false,
+    email:   false,
+    website: false
+};
+
+function subEvent (name, raiseAction) {
```

Comment:
It was an example name we need a better one.
Let's just use `raiseActionAsync` only for test purposes

Reply:
But in this case it's better to change arguments order `raiseAction, eventName`